PacketDotNet/IPv6ExtensionHeader.cs

```diff
@@ -0,0 +1,89 @@
+﻿using PacketDotNet.MiscUtil.Conversion;
+using PacketDotNet.Utils;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+/*
+This file is part of PacketDotNet
+
+PacketDotNet is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+PacketDotNet is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with PacketDotNet.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ * Copyright 2018 Steven Haufe<haufes@hotmail.com>
+ */
+
+namespace PacketDotNet
+{
+    [Serializable]
+    public class IPv6ExtensionHeader
+    {
+        public static HashSet<IPProtocolType> extensionHeaderTypes = new HashSet<IPProtocolType>{ IPProtocolType.HOPOPTS,
```

Comment:
I think this field can be in IPv6Header only as this is unused here (and be made private and named like _this.

Reply:
No it's used from in IPv6Packet

Reply:
Yeah, but only in IPv6Packet so it should probably be moved into that file (i.e., IPv6Packet.cs).